.github/workflows/xtest.yml

```diff
@@ -112,6 +123,7 @@ jobs:
           python-version: "3.12"
       - run: |-
           pip install -r scripts/requirements.txt
```

Comment:
Maybe just add it to the requirements.txt?

Reply:
Done